### PR TITLE
chore: refactoring of manual payload

### PR DESCRIPTION
**Pull request by hatskier**

### CODE REVIEW

**Review by cehali — APPROVED** [review]
LGTM

File: packages/evm-connector/src/wrappers/BaseWrapper.ts [hunk 1]
```diff
@@ -17,38 +11,34 @@ interface OverwriteFunctionArgs {
...
-    // will be padded right and contract will revert with `CalldataMustHaveValidPayload`
-    if (shouldBePreparedForManualUsage) {
-      const originalPayload = RedstonePayload.prepare(
-        signedDataPackages,
-        unsignedMetadata
-      );
+    const originalPayload = RedstonePayload.prepare(
+      signedDataPackages,
+      unsignedMetadata
+    );
```
Comment (cehali) [inline packages/evm-connector/src/wrappers/BaseWrapper.ts:39]: Why it is above `if (shouldBeMultipleOf32)`? We run  `prepare` function two times always

File: packages/evm-connector/src/wrappers/BaseWrapper.ts [hunk 2]
```diff
@@ -17,38 +11,34 @@ interface OverwriteFunctionArgs {
...
-  async getRedstonePayloadForManualUsage(
-    params?: ParamsForDryRunVerification
-  ): Promise<string> {
-    const shouldBePreparedForManualUsage = true;
-    const payloadWithoutZeroExPrefix = await this.getBytesDataForAppending(
-      params,
-      shouldBePreparedForManualUsage
-    );
+  async getBytesDataForAppending(): Promise<string> {
+    const shouldBeMultipleOf32 = false;
```
Comment (cehali) [inline packages/evm-connector/src/wrappers/BaseWrapper.ts:19]: I am not really a fan of it but I couldn't develop a better solution. The only thing I came up with is to pass it as an object to `prepareRedstonePayload` which looks then more like config than some random boolean argument

Comment (hatskier) [reply packages/evm-connector/src/wrappers/BaseWrapper.ts:19]: Frankly, I don't see a value of passing an object with only one prop

File: packages/evm-connector/src/wrappers/BaseWrapper.ts (hunk 1 above)
Comment (hatskier) [reply packages/evm-connector/src/wrappers/BaseWrapper.ts:39]: Pushed a fix

**Review by cehali — APPROVED** [review]

**Review by cehali — APPROVED** [review]

**Pull request merged**